src/Features/Core/Portable/Diagnostics/Analyzers/PreferFrameworkTypeDiagnosticAnalyzerBase.cs

```diff
@@ -111,37 +111,30 @@ protected void AnalyzeNode(SyntaxNodeAnalysisContext context)
             }
             // earlier we did a context insensitive check to see if this style was preferred in *any* context at all.
             // now, we have to make a context sensitive check to see if options settings for our context requires us to report a diagnostic.
-            if (ShouldReportDiagnostic(predefinedTypeNode, optionSet, language, out var diagnosticId, out var diagnosticSeverity))
+            if (ShouldReportDiagnostic(predefinedTypeNode, optionSet, language, out var descriptor, out var severity))
```

Comment:
Please annotate whether this is the effective severity or the default severity

Reply:
Default severity shouldn't exist as a concept in the code style analyzers.

Reply:
Update: The recent work to avoid public API changes eliminated the use of `ReportDiagnostic` to refer to the default severity across the board, so confusion should no longer be possible.